modules/cert-manager-remove-resources-console.adoc

```diff
@@ -27,14 +27,13 @@ Optionally, after uninstalling the {cert-manager-operator}, you can remove its r
 
 *** `Certificate`
 *** `CertificateRequest`
-*** `CertManager` (`config.openshift.io`)
 *** `CertManager` (`operator.openshift.io`)
 *** `Challenge`
 *** `ClusterIssuer`
 *** `Issuer`
 *** `Order`
 
-. Remove the `openshift-cert-manager-operator` namespace.
+. Remove the `cert-manager-operator` namespace.
 .. Navigate to *Administration* -> *Namespaces*.
-.. Click the Options menu {kebab} next to the *openshift-cert-manager-operator* and select *Delete Namespace*.
-.. In the confirmation dialog, enter `openshift-cert-manager-operator` in the field and click *Delete*.
+.. Click the Options menu {kebab} next to the *cert-manager-operator* and select *Delete Namespace*.
```

Comment:
I know this isn't your change, but should the Options menu be **Options** menu? It looks like it isn't bolded throughout the the doc.

Reply:
I think this is weird too, but this is how it's used throughout the doc set pretty consistently (`Options menu {kebab}`). It's not bolded because it doesn't actually say "Options" in the UI, but IMO then it shouldn't be capitalized either :shrug: But for consistency I'm keeping it as is for now.